### PR TITLE
Mobs no longer create blood trails while being thrown

### PR DESCRIPTION
**Pull request by BrumalLake**
## About The Pull Request
they get a little flag while being thrown that prevents the creation of trails
new `THROWN` flag that goes under `movement_type` for all moveables

## Why It's Good For The Game
trying to move bodies but it keeps making streaks

## Changelog
:cl:
add: throwing creatures doesn't make blood trails anymore
/:cl:

### CODE REVIEW

**Comment by yyzsong** [comment]
Mfw walk into medbay and see medical doctors chucking people across the fucking medbay to save time

**Review by qwertyquerty — APPROVED** [review]

**Review by qwertyquerty — CHANGES_REQUESTED** [review]
Actually, I take it back. If you're going to have a state flag for an object currently being thrown, it should be applied to all throwable objects, not just mobs

**Comment by ivanmixo** [comment]
So you can just throw someone over and over down the hall and they won't lose any blood? This makes 0 sense.

**Comment by BrumalLake** [comment]
> 
> 
> This makes 0 sense.

feel like this is gonna be your recurring joke after i did it to you

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -659,7 +659,7 @@
 		makeTrail(newloc, T, old_direction)
 
 /mob/living/proc/makeTrail(turf/target_turf, turf/start, direction)
-	if(!has_gravity())
+	if(!has_gravity() || movement_type & THROWN)
```
Comment (PowerfulBacon) [inline code/modules/mob/living/living.dm]:
```suggestion
	if(!has_gravity() || (movement_type & THROWN))
```
Just to be safe I would do something like this, bit checks not in brackets can get funky

**Comment by BrumalLake** [comment]
should the flag be applied to all moveables when you throw them right now? i don't think anyone will use the flag any time soon on non mobs

**Comment by qwertyquerty** [comment]
Yes, it should be applied to anything that can possibly be thrown.

**Comment by Isy232** [comment]
People still bleed when thrown in anything even remotely realistic...

**Comment by BrumalLake** [comment]
> People still bleed when thrown in anything even remotely realistic...

holy shit stop talking about realism and even then you lose the realism argument because yes you obviously still bleed and if you paid attention you'd see you can still drip blood but you don't drag against the ground while flying through the air 🤯

**Comment by BrumalLake** [comment]
> 
> 
> Yes, it should be applied to anything that can possibly be thrown.

addressed

**Review by qwertyquerty — APPROVED** [review]

**Comment by KubeRoot** [comment]
I don't like this very much so, not when the argument for it is that it shouldn't be making blood streaks when *moving bodies*. There are multiple ways to move bleeding bodies without leaving blood trails, including fireman carry, rollerbeds and bodybags, literally yeeting people across rooms shouldn't be better than those options.

**Comment by Sinestia** [comment]
Caveman PR

**Comment by BrumalLake** [comment]
meant for moving bodies a couple of tiles not long distances
they still streak while landing

**Comment by chizzy376** [comment]
> meant for moving bodies a couple of tiles not long distances
> they still streak while landing

This PR coupled with the one that allows you to catch thrown bodies will have medical staff play basketball with the dying person's body since that would be the most efficient way to move people around. Besides, hurling someone that is bleeding at mach 5 is going to leave a blood trail, since they're still bleeding and the blood in the air that leaks out will eventually fall to the ground.. thus forming a blood trail on the ground when thrown. I really do not like this PR since you'll turn rescuing from a fireman carry to a game of professional basketball

**Comment by BrumalLake** [comment]
i did not ask for that to get merged and it'll be faster to carry or use other methods in long distances
they will still leave some blood trails

>they're still bleeding and the blood in the air that leaks out will eventually fall to the ground

you just described dripping

**Comment by ivanmixo** [comment]
make mobs do a big blood splatter on the tile they land on

**Pull request merged**